### PR TITLE
fix:improve styles to display the status of PR

### PR DESCRIPTION
**Pull request by hardikmodi1**
#425 
Screenshot 2020-10-06 at 8 01 46 PM

### CODE REVIEW

**Comment by jenkoian** [comment]
Thanks, are you able to include a screenshot?

**Comment by hardikmodi1** [comment]
@jenkoian added screenshot and updated the PR to resolve [this comment](https://github.com/jenkoian/hacktoberfest-checker/pull/431#issuecomment-703275060)

File: src/pages/User/components/PullRequests/PullRequest/PullRequestInfo.js [hunk 1]
```diff
@@ -7,7 +7,11 @@ const PullRequestInfo = ({ pullRequest }) => (
       <span className="text-light-blue">
         {pullRequest.repo_name}#{pullRequest.number}
       </span>
-      {pullRequest.is_pending && <em className="text-mid-grey"> Pending</em>}
+      {pullRequest.is_pending && (
+        <div className="ml-4 rounded-full px-2 text-white bg-mid-blue inline text-xs">
+          <em>Pending</em>
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/PullRequest/PullRequestInfo.js:12]: I think get rid of the `<em>` (can use `<span>` or something) and lowercase the `p`

**Comment by hardikmodi1** [comment]
Updated the screenshot

**Review by jenkoian — APPROVED** [review]
Thanks @hardikmodi1

**Pull request merged**